sources/src/main/java/com/google/solutions/jitaccess/web/RuntimeEnvironment.java

```diff
@@ -92,10 +93,18 @@ private static HttpResponse getMetadata(String path) throws IOException {
     }
   }
 
-  private boolean isRunningOnAppEngine() {
+  public boolean isRunningOnAppEngine() {
     return System.getenv().containsKey("GAE_SERVICE");
   }
 
+  public boolean isRunningOnCloudRun() {
+    return System.getenv().containsKey("CLOUD_RUN");
```

Comment:
Not sure if we should rely on `CLOUD_RUN`, it's not documented here: https://cloud.google.com/run/docs/container-contract. It might be better to check for `K_SERVICE`.